index.html

```diff
@@ -93,258 +96,275 @@ <h1 class="title">
-            </h2>
-            <ul class="span-grid">
-              <li class="color">CANOPY</li>
-              <img
-                src="./resources/work-section/icons/Counterdot.png"
-                alt=""
-                class="dot"
-              />
-              <li>Back End Dev</li>
-              <img
-                src="./resources/work-section/icons/Counterdot.png"
-                alt=""
-                class="dot"
-              />
-              <li>2015</li>
-            </ul>
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <ul class="span-grid2">
-              <li>&nbsp&nbsp&nbsp HTML</li>
-              <li>&nbsp&nbsp&nbsp CSS</li>
-              <li>&nbsp&nbsp&nbsp&nbsp JavaScript</li>
-            </ul>
-            <button class="button-project">See Project</button>
+            <div class="Desktop-version">
+              <h2 class="general-titles">
+                Multi-Post <br />
+                Stories
+              </h2>
+              <ul class="span-grid">
+                <li class="color">CANOPY</li>
+                <img
+                  src="./resources/work-section/icons/Counterdot.png"
+                  alt=""
+                  class="dot"
+                />
+                <li>Back End Dev</li>
+                <img
+                  src="./resources/work-section/icons/Counterdot.png"
+                  alt=""
+                  class="dot"
+                />
+                <li>2015</li>
+              </ul>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="span-grid2">
+                <li>&nbsp&nbsp&nbsp HTML</li>
+                <li>&nbsp&nbsp&nbsp CSS</li>
+                <li>&nbsp&nbsp&nbsp&nbsp JavaScript</li>
+              </ul>
+              <button class="button-project">See Project</button>
+            </div>
           </div>
         </div>
       </section>
     </main>
     <footer class="footer">
       <div class="footer-container">
-        <h2>About Myself</h2>
-        <p class="presentation">
-          I’m a software developer! I can help you build a product , feature or
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hesitate to contact
-          me.
-        </p>
-        <a class="connection" href="index.html">LET’S CONNECT</a>
-        <ul class="span">
-          <li>
-            <a href="https://twitter.com/">
-              <img
-                src="./resources/icons/Vectoricons-3.svg"
-                alt="Twitter"
-                class="social-media"
-              />
-            </a>
-          </li>
+        <div class="first-part-footer">
+          <div class="about-myself">
+            <h2>About Myself</h2>
```

Comment:
- [ ] According to the project requirement please I would kindly suggest you to stick to the figma design kindly fix the arrow button make align to each other

- [ ] Also I would kindly suggest you to work on the get in touch button to make it fit exact like figma template design kindly fix it 

- [ ] your design 
![Screenshot_20230506-114702](https://user-images.githubusercontent.com/92458236/236620797-fb06eaab-6e9d-49b5-90fc-660414f7d96f.jpg)

Reply:
> - [ ] Good job @SaveryIV👍, As requested by the previous code reviewer, Please kindly align your categories icons to be in the same position. Aligning category icons in the same position is important because it helps maintain consistency and coherence in the design. When icons are not aligned, it can create visual dissonance, making the design appear messy and disorganized. Keeping icons in the same position as specified in the Figma design helps ensure that the design is faithful to the original design intent and maintains a professional and polished appearance. This can help to enhance the user experience and create a more positive impression of the website or application.😊
> 
> Figma design👇
> 
> ![ico-figma](https://user-images.githubusercontent.com/40009354/236633972-4dd8a82c-e4a2-4748-9ff7-b3b73d84fbcc.png)
> 
> Your design👇
> 
> ![ico](https://user-images.githubusercontent.com/40009354/236633981-f0184d26-1e09-485b-ae9c-6e4a8acbb775.png)
>